### DISCUSSION TITLE
Congrats on the release of eCommerce!

### DISCUSSION THREAD

**Discussion by Martleby** [tailwindlabs/tailwindcss#5190] [Kind words]
I knew it was going to be good, so I snapped it up!

I don't have a project in mind for it at the moment - but I buy Tailwind stuff to support you guys and the amazing work you do.

PS Are you going to open a merch store at any point? I could surely rock a 1 pint Tailwind-themed coffee mug :)

**Comment by ghost** [comment]
I LOVE TAILWIND :)

**Comment by simonswiss** [comment]
Thank you for the support and the kind words! ❤️

Not sure about the merch store, but for now there are the Tailwind CSS T-shirts on Cotton Bureau:

https://cottonbureau.com/people/tailwind-labs

**Comment by ghost** [comment]
Tailwind is so good thank you !
